src/components/Shell/_mixins.scss

```diff
@@ -103,22 +103,34 @@
   &__skip-to-content {
     display: flex;
     position: relative;
-    height: 0;
```

Comment:
Please note that I just removed this height rule, based on guidance about ensuring that the "hidden" content doesn't actually have a zero height: https://webaim.org/techniques/css/invisiblecontent/

We want the child of this element, the actual link, to be readable by a screenreader.